pwa/src/components/applications/applicationForm.tsx

```diff
@@ -0,0 +1,233 @@
+                            <div className="col-6">
+                              {application !== null && application.secret !== null ? (
+                                <GenericInputComponent type={"text"} name={"secret"} id={"secretInput"}
+                                                       data={application.secret} nameOverride={"Secret"} />
+                              ) : (
+                                <GenericInputComponent type={"text"} name={"secret"} id={"secretInput"}
+                                                       nameOverride={"Secret"} />
+                              )}
+                            </div>
+                          </div>
+                          <div className="row">
+                            <div className="col-6">
+                              {application !== null && application.resource !== null ? (
+                                <GenericInputComponent type={"text"} name={"resource"} id={"resourceInput"} data={application.resource}
+                                                       nameOverride={"Resource"} />
+                              ) : (
+                                <GenericInputComponent type={"text"} name={"resource"} id={"resourceInput"}
+                                                       nameOverride={"Resource"} />
+                              )}
+                            </div>
+                          </div>
+
+                          <Accordion
+                            id="applicationAccordion"
+                            items={[
+                              {
+                                title: "Domains",
+                                id: "domainsAccordion",
+                                render: function () {
+                                  return (
+                                    <>
+                                      {application !== null && application.domains !== null ? (
+                                        <ArrayInputComponent
+                                          id={"domains"}
+                                          label={"Domains"}
+                                          data={application.domains}
+                                        />
+                                      ) : (
+                                        <ArrayInputComponent
+                                          id={"domains"}
+                                          label={"Domains"}
+                                          data={null}
+                                        />
+                                      )}
+                                    </>
+                                  );
+                                },
+                              }
+                            ]}
+                          />
+                        </>
+                      )}
+                    </div>
+                  </div>
+                )
+              }} />
+      </form></>
```

Comment:
Ik ben zelf geen fan van overal de `<> </>` omheen zetten. Zou liever een default van bijvoorbeeld een `section` of `div` o.i.d. zien.